lib/msal-node/src/network/HttpClient.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { INetworkModule, NetworkRequestOptions } from '@azure/msal-common';
+import { HttpMethod } from './../utils/Constants';
+import axios from 'axios';
+
+/**
+ * This class implements the Fetch API for GET and POST requests. See more here: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
```

Comment:
This comment is incorrect, as fetch is not used in this. I would say something more along the lines of "implements methods for making network requests"

Reply:
Thanks. Sorry missed this :). Corrected now.